starter-code/scripts/article.js

```diff
@@ -18,6 +18,7 @@ Article.prototype.toHtml = function() {
     $newArticle.addClass('draft');
   }
   $newArticle.attr('data-category', this.category);
+  $newArticle.attr('data-author', this.author);
```

Comment:
Just to clarify, if you had used `.data` instead of `.attr`...
this is the getter `$newArticle.data('data-author');`
this is the setter `$newArticle.data('data-author', this.author);`